src/3rd_party-static/gmock-1.7.0/src/gmock-spec-builders.cc

```diff
@@ -761,7 +761,7 @@ bool Mock::AsyncVerifyAndClearExpectations(int timeout_msec)
   return AsyncVerifyAndClearExpectationsLocked(timeout_msec);
 }
 
-bool Mock::AsyncVerifyAndClearExpectationsLocked(const int timeout_msec_in)
+bool Mock::AsyncVerifyAndClearExpectationsLocked(int timeout_msec)
```

Comment:
@LuxoftAKutsan, 
In case of final solution, when we wait all this amount of `msecs`, it's correct way to use only this variable and reduce it directly.